Content.Server/Bible/BibleSystem.cs

```diff
@@ -0,0 +1,97 @@
+using System;
+using Robust.Shared.GameObjects;
+using Content.Shared.Interaction;
+using Content.Shared.Inventory;
+using Content.Shared.MobState.Components;
+using Content.Shared.Damage;
+using Content.Shared.Popups;
+using Content.Server.Cooldown;
+using Content.Server.Inventory;
+using Content.Server.Mind.Components;
+using Content.Server.Bible.Components;
+using Content.Server.Popups;
+using Robust.Shared.IoC;
+using Robust.Shared.Random;
+using Robust.Shared.Audio;
+using Robust.Shared.Player;
+using Robust.Shared.Localization;
+using Robust.Shared.Timing;
+
+
+namespace Content.Server.Bible
+{
+    public class BibleSystem : EntitySystem
+    {
+        public override void Initialize()
+        {
+            base.Initialize();
+
+            SubscribeLocalEvent<BibleComponent, AfterInteractEvent>(OnAfterInteract);
+        }
+
+        private void OnAfterInteract(EntityUid uid, BibleComponent component, AfterInteractEvent args)
+        {
+            var invSystem = EntitySystem.Get<InventorySystem>();
+            var random = IoCManager.Resolve<IRobustRandom>();
+            var currentTime = IoCManager.Resolve<IGameTiming>().CurTime;
+            EntityManager.TryGetComponent<MetaDataComponent>(uid, out var meta);
```

Comment:
every entity has a metadata, also can just use the proxy methods
```suggestion
            var meta = MetaData(uid);
```